sambacc/samba_cmds.py

```diff
@@ -36,6 +36,14 @@ def get_samba_specifics() -> list:
     return out
 
 
+def smbd_stdout_opt() -> str:
```

Comment:
I doubt we'll need this function outside of samba_cmds.py let's prefix it with an underscore (_smbd_stdout_opt) to make it "private".